### ISSUE TITLE
Facebook UI is broken in multiple ways

### ISSUE THREAD

**Issue by eppfel** [secure-software-engineering/rose#264]
**Update:** Missing easing function in semantic-ui nag module stops code execution.

---
I don't know, if this is due to the changes in 00278fe27b56832897480b7a88b1476a7ece2a15, but the 
- [ ] comment labels do not work anymore and
- [ ] MutationObserver fails to add more labels   

... in the current stage of the master branch.

First I though the "status-update" extractor is broken, but it successfully extracts a contentId on "like-content" interactions. Sadly there is no error, but I could trace back the script crashing in https://github.com/secure-software-engineering/rose/blob/master/app/rose/extractor-engine.js#L70 so far.

**Comment by iamolivinius** [comment]
Most likely related to this: https://github.com/secure-software-engineering/rose/issues/259#issuecomment-207565742

Will be fixed when semantic-ui 2.1.9 is released.

**Comment by dharma-guardian** [comment]
How so? I see no relation to semantic-ui here.

**Comment by iamolivinius** [comment]
Mystery challenge :wink:

**Comment by iamolivinius** [comment]
As far as I remember: Missing easing function stops code execution (runtime error)

**Comment by dharma-guardian** [comment]
Did not realize you fixed it in Semantic-Org/Semantic-UI#3862 . A quick test with excluding the nag module proved everything works fine. Will leave it open, as it is still broken until the semantic-ui update.

**Comment by eppfel** [comment]
A temporary fix, to work on the content script is to manually add the fix to facebook-ui.js

```JS
//quickfix to prevent semantic-ui nag module killing code execution
$.extend( $.easing, {
  easeOutQuad: function (x, t, b, c, d) {
    return -c *(t/=d)*(t-2) + b;
  }
});
```
